### ISSUE TITLE
instances dct mask head

### ISSUE THREAD

**Issue by maralzar** [aliyun/DCT-Mask#2]
I initial on my dataset but this is always true and the output of instances doesn't have GT.
for instances_per_image in instances:
            if len(instances_per_image) == 0:
                   continue
#OUTPUT INSTANCES:
instances [Instances(num_instances=0, image_height=768, image_width=1151, fields=[proposal_boxes: Boxes(tensor([], device='cuda:0', size=(0, 4))), objectness_logits: tensor([], device='cuda:0'), gt_classes: tensor([], device='cuda:0', dtype=torch.int64), gt_boxes: Boxes(tensor([], device='cuda:0', size=(0, 4))), gt_masks: PolygonMasks(num_instances=0)]), Instances(num_instances=0, image_height=768, image_width=1159, fields=[proposal_boxes: Boxes(tensor([], device='cuda:0', size=(0, 4))), objectness_logits: tensor([], device='cuda:0'), gt_classes: tensor([], device='cuda:0', dtype=torch.int64), gt_boxes: Boxes(tensor([], device='cuda:0', size=(0, 4))), gt_masks: PolygonMasks(num_instances=0)])]
